src/main/resources/graphql/game.graphqls

```diff
@@ -12,8 +12,38 @@ type Game {
     visitorTeam: Team!
 }
 
+input GameInput {
+    id: ID!,
+    date: String!,
+    homeTeamScore: Int!,
+    visitorTeamScore: Int!,
+    season: Int!,
+    period: Int!,
+    status: String!,
+    time: String!,
+    postseason: Boolean!,
+    homeTeam: TeamInput!,
```

Comment:
I think here a team ID would be better suited, so we don't need to pass a whole Team object (actually that's the only part we use)

Reply:
Done! 🔥